Gemfile

```diff
@@ -7,19 +7,17 @@ end
 
 ruby "2.4.1"
 
-gem "autoprefixer-rails"
+gem "bcrypt"
 gem "delayed_job_active_record"
-gem "flutie"
 gem "honeybadger"
 gem "jquery-rails"
+gem "omniauth"
+gem "omniauth-facebook"
 gem "pg"
 gem "puma"
-gem "rack-canonical-host"
 gem "rails", "~> 5.1.3"
-gem "recipient_interceptor"
```

Comment:
That's a good gem to have... but up to you...

Reply:
Thanks for the feedback Elle, I will create .rubocop.yml and use it for the overrides. I thought maybe I'd keep enable and disable as a guide to what things I can change but I guess I can get rid of them and look them up as needed.